packages/image/src/__test__/load.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { type ImageSource, load } from "..";
+
+class MockImage {
+  public src = "";
+
+  public setAttribute(name: string, value: string) {
+    this[name] = value;
+  }
+
+  public constructor() {
+    MockImage.lastInstance = this;
+  }
+
+  public static lastInstance: MockImage;
+}
+
+describe("load function", () => {
+  const originalImage = window.Image;
+
+  beforeAll(() => {
+    window.Image = MockImage as unknown as typeof Image;
+  });
+
+  afterAll(() => {
+    window.Image = originalImage;
+  });
+
+  it("should load defaultSrc when webpSrc is not provided", () => {
+    const imageSource: ImageSource = {
+      defaultSrc: "./images/test.png",
+    };
+
+    load([imageSource]);
```

Comment:
We can remove type import

```suggestion
    load([{ defaultSrc: "./images/test.png" }]);
```

Reply:
[ec4651d](https://github.com/fepack/image/pull/32/commits/ec4651d56b32d43392d22b2b808c37f966a7d33d) 😁